sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -288,19 +289,21 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
        * Note, currently the new table creation by this API doesn't fully cover the V2 table.
        * TODO (SPARK-33638): Full support of v2 table creation
        */
-      val cmd = CreateTableStatement(
-        originalMultipartIdentifier,
+      val cmd = CreateV2Table(
+        UnresolvedDBObjectName(
+          originalMultipartIdentifier,
+          isNamespace = true),
```

Comment:
why is it `true`? i guess it is a `table` name

Reply:
This should be false.